contributing/webapp.md

```diff
@@ -471,37 +493,100 @@ You can access the application by running it from IntelliJ. The url is:
+```
+pauline@ThinkPad-T14s-Gen-6:~/Workspace/rudder/webapp/sources/rudder/rudder-web/target$ ls
+classes                       rudder-web-8.3.5-SNAPSHOT              surefire-reports
+classes.-287291424.timestamp  rudder-web-8.3.5-SNAPSHOT-classes.jar  test-classes
+generated-sources             rudder-web-8.3.5-SNAPSHOT-tests.jar    test-classes.64873501.timestamp
+generated-test-sources        rudder-web-8.3.5-SNAPSHOT.war
+maven-status                  specs2-reports
+```
+The war generated is named `rudder-web-8.3.5-SNAPSHOT.war` in this example.
+
+Copy the war in the vbox server
+```
+cd <workspace>/rudder-tests
+vagrant upload <workspace>/rudder/webapp/sources/rudder/rudder-web/target/rudder-web-8.3.5-SNAPSHOT.war
+```
+
+Connect to the vbox and put the war in the share directory and restart the service `rudder-jetty`
+```
+vagrant ssh myvmtest_server
+sudo su
+mv rudder-web-8.3.5-SNAPSHOT.war /opt/rudder/share/webapps/
+mv rudder.war rudder.war.save
+mv rudder-web-8.3.5-SNAPSHOT.war rudder.war
+systemctl restart rudder-jetty
+```
+Get the ip of the server by running `ip a` and check the change in the browser for instance: `http://192.168.49.2` (user admin/admin)
 
-> Important note: This script does not setup Intellij ([Setup workspace development with IntelliJ and Maven](#part-2---setup-workspace-development-with-intellij-and-maven)) 
- and Apache Directory Studio ([LDAP connection](#test-ldap-connection)). These still have to be set manually.
+#### Testing rudder in production mode
 
-> To start Apache Directory Studio: `/opt/ApacheDirectoryStudio/ApacheDirectoryStudio`
+Some use cases require an environment in production mode, like ticket validation for instance. Run an environment matching the ticket. Validating ticket require to install a vm with rudder in the same version required in the ticket. 
 
-> Important note: this script should only be ran once. Every other startup, only do the following command: `./rtf platform setup debian9_dev`
+If a pull request is merged recently, then you need to test on a SNAPSHOT version, and you need to wait the next nightly build after the merge. Make sure you're using the last nightly build. You can check the date of the last logs `/var/log/rudder/webapp/webapp.log` and if you need to update to the last version you can run `apt update && apt install rudder-server` from the vbox server.
+Example of configuration using nightly versions :
+```
+{
+"default": { "rudder-version": "8.3-nightly", "system": "debian12", "inventory-os": "debian" },
+"server":  { "rudder-setup": "server" }
+}
+```
 
-> All rudder repos will be cloned (including rtf and ncf) in `/home/<user>/rudder/`
+See the versions of rudder here https://repository.rudder.io/
 
-> Disclaimer: this script might not work on your machine. If it does not, it still is a good guideline when trying to setup a dev environment since it traces every single step of this readme + rtf setup readme
+#### Testing rudder api
```

Comment:
This seems to be a way to use rudder api in all cases, which is a documented process that is the same as our users.

Maybe you wanted to tell how to run an api request to you env server, but then it shouldn't be the vm ip but localhost:port. (and tip suggested by @clarktsiory is what you should do, unless your validating another user/rights case